src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -974,6 +977,12 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                 // Any errors generated below here are considered Emit diagnostics 
                 // and will not be reported to callers Compilation.GetDiagnostics()
 
+                ImmutableArray<SourceSpan> dynamicAnalysisSpans = ImmutableArray<SourceSpan>.Empty;
+                if (_moduleBeingBuiltOpt.EmitOptions.EmitDynamicAnalysisData)
+                {
+                    flowAnalyzedBody = Instrumentation.InjectInstrumentation(methodSymbol, flowAnalyzedBody, methodOrdinal, compilationState, _compilation, _diagnostics, _debugDocumentProvider, out dynamicAnalysisSpans);
```

Comment:
From my discussion with Neal this morning, I thought we do most of the local lowering in a single scan through the BoundNodes. @gafter, it is ok to have this separate step?